src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -4,24 +4,11 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.common.base.Objects;
+import com.google.common.base.Optional;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.util.Arrays;
-import org.stellar.sdk.xdr.ClaimableBalanceID;
-import org.stellar.sdk.xdr.ClaimableBalanceIDType;
-import org.stellar.sdk.xdr.DecoratedSignature;
-import org.stellar.sdk.xdr.EnvelopeType;
-import org.stellar.sdk.xdr.Hash;
-import org.stellar.sdk.xdr.Int64;
-import org.stellar.sdk.xdr.OperationID;
-import org.stellar.sdk.xdr.SequenceNumber;
-import org.stellar.sdk.xdr.TransactionEnvelope;
-import org.stellar.sdk.xdr.TransactionSignaturePayload;
-import org.stellar.sdk.xdr.TransactionV0;
-import org.stellar.sdk.xdr.TransactionV0Envelope;
-import org.stellar.sdk.xdr.TransactionV1Envelope;
-import org.stellar.sdk.xdr.Uint32;
-import org.stellar.sdk.xdr.XdrDataOutputStream;
+import org.stellar.sdk.xdr.*;
```

Comment:
wildcard imports are discouraged, if using IntelliJ for IDE, can set that to format/sort the imports automatically by using a hot key, hopefully `*` isn't used elsewhere, do you mind adding that as a `Guidelines` item in CONTRIBUTING.md?

Reply:
Fixed in a635bc35f5d55a0aa710ba92f01ca4961c9ea577.

For guidance, let's improve it in another PR. I have created an issue #493.

Reply:
yes, great approach, thanks!